bitnami/zookeeper/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: zookeeper
-version: 1.6.1
+version: 1.7.0
```

Comment:
This is a fix, you're not introducing any new parameter or functionality. Therefore, I'd increase the "patch version":

```diff
- version: 1.6.1
+ version: 1.6.2
```

Reply:
Totally agree